guidelines.md

```diff
@@ -1,84 +1,142 @@
+* Resource Requests and Limits: Define realistic resource requests and limits for all
+  workloads. For resource-intensive examples, clearly document these and, if feasible,
+  offer scaled-down versions for resource-constrained environments.
+* External Links in Manifests: All URLs used in manifests must point to reliable, versioned sources.
+* Container Images:
+  * Publicly Accessible Images: Images used in examples MUST be publicly accessible
+    from well-known, reputable container registries.
+  * Image Tagging: Images MUST be tagged with a specific version instead of using `:latest` tags.
+  * If an example requires a custom-built image not available on public registries, the
+    `Dockerfile` and all necessary source files (build context) MUST be included within
+    the example's subdirectory (e.g., in an `image/` subfolder).
+
+### Documentation and Commands
+
+* Linking to Official Docs: On the first mention of a Kubernetes concept, link to its
+  official documentation page.
+* Code Highlighting: Use appropriate code highlighting for shell commands and YAML
+  manifests, consistent with GitHub's rendering (typically Rouge-supported types).
+    * Commands to be copied by the user should use the `shell` syntax highlighting type,
+      and do not include any kind of prompt characters (e.g., `$`, `#`).
+    * Example output should be in a separate block to distinguish it from the commands.
+* Placeholders: When providing commands or configuration where users need to substitute
+  their own values, use angle brackets: e.g., `<YOUR_NAMESPACE>`, `<YOUR_BUCKET_NAME>`.
+* Screenshots/Diagrams (Optional but helpful): For complex examples, screenshots of the
+  running application or simple architecture diagrams can be very beneficial. Store these
+  within the example's subdirectory. Asciinema recordings for terminal interactions are
+  also welcome.
+
+## Maintenance and Lifecycle
+
+* Review and Updates: Examples are expected to be kept reasonably up-to-date with current
+  Kubernetes versions and best practices. Contributors are encouraged to revisit their
+  examples periodically. The `README.md` should specify the Kubernetes version(s) against
+  which the example was last tested.
+* Archiving Policy:
+    * A process will be established by SIG Apps for identifying outdated, unmaintained, or
+      broken examples.
+    * Such examples may be moved to an `archive/` directory or removed after a deprecation
+      period and attempts to find new maintainers. This ensures the repository remains a
+      reliable source of current best practices.
+    * The `README.md` of archived examples should clearly state their archived status and
+      potential incompatibility with current Kubernetes versions.
+* Community Maintenance: SIG Apps will act as the overall steward, and individual example
+  maintainers (original authors or new volunteers) are crucial for the health of the repository.
 
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
```

Comment:
Is this still used or should I remove it?

Reply:
For removal.